### ISSUE TITLE
nanoff command unable to find ESP32_WROOM_32 firmware in repository

### ISSUE THREAD

**Issue by Kleebz** [nanoframework/Home#753]
Details:
When attempting to flash the firmware, Error  code E9005 is displayed. 
"Error E9005: Can't find the target in the Bintray repository."

Steps to reproduce:
1. Open the Package Manager Console in Visual Studio 2019
2. Install nanoFirmwareFlasher : "dotnet tool install -g nanoFirmwareFlasher"
3. Update nanoFirmwareFlasher  : "dotnet tool update -g nanoFirmwareFlasher"
4. Attempt to install the firmware : "nanoff --update --target ESP32_WROOM_32 --serialport COM3 --stable -v diag" 
![Error_E9005](https://user-images.githubusercontent.com/29114697/118383246-0143ee00-b5b1-11eb-8bc0-ac11908bbfb0.PNG)

**Comment by josesimoes** [comment]
You're using an outdate version of nanoff. Please update.

**Comment by AbhiAbzs** [comment]
I was also facing the same issue. The error message was
```
Trying to find ESP32_WROOM_32 in community targets repository...Error E9005
```
I have tired updating as well but that also didn't worked.
The whole confusion with updating the _nanoFramework_ Flasher was due to the fact that the earlier method for installing & updating the nanoFramework flashed was
```pwsh
dotnet tool install -g nanoFirmwareFlasher
dotnet tool update -g nanoFirmwareFlasher
```
using which I was getting this issue
![image](https://user-images.githubusercontent.com/29309508/118539569-309f5a00-b76d-11eb-9a95-0c529b26c547.png)
![image](https://user-images.githubusercontent.com/29309508/118541367-37c76780-b76f-11eb-8604-79b977ae6e73.png)

I just uninstalled that using bellow command & that installed the _nanoFramework_ using the next command which was from its documentation & then it just worked (you would be required to press the boot button).
```pwsh
dotnet tool uninstall -g nanoFirmwareFlasher
dotnet tool install -g nanoff
```
![image](https://user-images.githubusercontent.com/29309508/118541154-0353ab80-b76f-11eb-9afd-1628aacb24c5.png)
Press boot button on esp32 board if you receive this kind of error.
![image](https://user-images.githubusercontent.com/29309508/118541621-896ff200-b76f-11eb-9278-b649516d357f.png)

Just thought of adding this for the sake of reference if any body else gets stuck. 😅
